rust/connlib/snownet/src/node.rs

```diff
@@ -1165,6 +1159,15 @@ fn add_local_candidate<TId>(
 ) where
     TId: fmt::Display,
 {
+    // srflx candidates don't need to be added to the local agent because we always send from the `base` anyway.
```

Comment:
What about peer reflexive? isn't that the same?

Reply:
Peer reflexive are only ever generated _within_ the agent and should never be added explicitly.